sass/components/_sponsors.scss

```diff
@@ -0,0 +1,63 @@
+.sponsors {
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+
+    &:not(:last-child) {
+        margin-bottom: 56px;
+    }
+
+    &__title {
+        font-size: 2.2rem;
+        margin: 0 0 24px;
+    }
+
+    &__content {
```

Comment:
Haha I've been webdeving for a long time and I've never used this pattern. Whats going on here / whats the name of this feature?

Reply:
Ahh the `&` is a sass feature.

Reply:
Oh, yeah, I wasn't sure what you were referring to.

Reply:
It was a combination of both things. I've havent used bem before either. Thanks for the info!